crates/aptos-crypto/benches/poseidon_bn254.rs

```diff
@@ -20,11 +20,36 @@ fn bench_group(c: &mut Criterion) {
 
     for n in 1..=16 {
         poseidon_bn254_ark(&mut group, n);
+        poseidon_bn254_neptune(&mut group, n);
     }
 
     group.finish();
 }
 
+fn poseidon_bn254_neptune<M: Measurement>(g: &mut BenchmarkGroup<M>, n: usize) {
+    let mut rng = thread_rng();
+
+    g.throughput(Throughput::Elements(n as u64));
+
+    g.bench_function(BenchmarkId::new("poseidon-bn254-neptune", n), move |b| {
+        b.iter_with_setup(
+            || {
+                (0..n)
+                    .map(|_| {
+                        let bytes = random_bytes(&mut rng, n);
+                        ark_bn254::Fr::from_le_bytes_mod_order(bytes.as_slice())
```

Comment:
Why is Fr being used here instead of AltFr if neptune is being benchmarked?

Reply:
The input of the hash function we want to implement should still be an `Fr`. Just the intermediate input & output to/from `Neptune` are `AltFr`s.